src/engine/engine.cpp

```diff
@@ -434,23 +434,23 @@ const char* DivEngine::getSystemName(DivSystem sys) {
     case DIV_SYSTEM_VRC6:
       return "Konami VRC6";
     case DIV_SYSTEM_OPLL:
-      return "Yamaha OPLL";
+      return "Sega FM Sound Unit";
     case DIV_SYSTEM_FDS:
       return "Famicom Disk System";
     case DIV_SYSTEM_MMC5:
-      return "MMC5";
+      return "Nintendo MMC5";
     case DIV_SYSTEM_N163:
       return "Namco 163";
     case DIV_SYSTEM_OPN:
       return "NEC PC-9801-26K";
     case DIV_SYSTEM_PC98:
-      return "NEC PC-9801-86";
+      return "PC-9801-86 + Chibi-oto";
```

Comment:
Hmmmm... wasn't the 86 board just a 2608?

Reply:
2608 without ADPCM + 86-PCM stereo Burr Brown DAC.